store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -64,7 +64,8 @@
   private Map<String, String> options;
   private String taskNo;
   private int localDictionaryThreshold;
-  private boolean isLocalDictionaryEnabled;
+  private boolean isLocalDictionaryEnabled = Boolean.parseBoolean(
```

Comment:
So what about the `localDictionaryThreshold`? To keep the same behavior, each configurations should be assigned the default value while init.

Reply:
it is already handled in below method when isLocalDictionaryEnabled is true
org.apache.carbondata.core.metadata.schema.table.TableSchemaBuilder#build